examples/gno.land/p/demo/memeland/helper.gno

```diff
@@ -0,0 +1,9 @@
+package memeland
+
+import (
+	"std"
+)
+
+func prevRealmIsOwner(addr std.Address) bool {
```

Comment:
the name doesn't match the implementation.

i think the readability would be improve by not adding this file.

Reply:
i will move this func then remove this file